### ISSUE TITLE
Pagamento sem CPF ou para não-residentes no Brasil

### ISSUE THREAD

**Issue by rodolphoeck** [pythonbrasil/pythonbrasil11#13]
As duas formas atuais de pagamento de inscrição (boleto ou MercadoPago) exigem CPF.
No caso de cartão de crédito, o CPF precisa estar associado ao cartão de crédito (não basta possuir um CPF válido).

Seria possível oferecer uma forma de pagamento sem exigir CPF? Transferência bancária? Wire-transfer para quem está no exterior?

Obrigado!

**Comment by filipecifali** [comment]
O paypal é algo viável não?

2015-08-26 20:27 GMT-03:00 Rodolpho Eckhardt notifications@github.com:

> As duas formas atuais de pagamento de inscrição (boleto ou MercadoPago)
> exigem CPF.
> No caso de cartão de crédito, o CPF precisa estar associado ao cartão de
> crédito (não basta possuir um CPF válido).
> 
> Seria possível oferecer uma forma de pagamento sem exigir CPF?
> Transferência bancária? Wire-transfer para quem está no exterior?
> 
> Obrigado!
> 
> —
> Reply to this email directly or view it on GitHub
> https://github.com/pythonbrasil/pythonbrasil11/issues/13.

## 

[ ]'s

Filipe Cifali Stangler

**Comment by rodolphoeck** [comment]
Sim. Se a APyB tem uma conta PayPal, posso fazer a transferência direto para ela também.

**Comment by renzon** [comment]
PayPal da APyB: cobranca@python.org.br. Infelizmente vamos ter que contornar enviando código de desconto e recebendo via Paypal. A não ser que alguém tenha ideia melhor. @rodolphoeck  pode fazer a transferencia por lá. Vou te enviar o código de desconto por email.
